src/components/pages/public/forgot-password/validate.js

```diff
@@ -1,8 +1,14 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
 
 export default function validate(values) {
     const errors = {};
     if (!values.email) {
-        errors.email = 'Looks like you forget your email.';
+        errors.email = (
+            <FormattedMessage
+                id="form.error.email.required"
+                defaultMessage="Looks like you forget your email." />
```

Comment:
here in validate function put it in a constant like
const EMAIL_ERROR = (
    <FormattedMessage
        id="form.error.email.required"
        defaultMessage="Looks like you forget your email." />
);